app/views/pulitzer/versions/_edit.html.erb

```diff
@@ -39,18 +39,21 @@
       <% end %>
     </div>
   </div>
-
   <% version.all_elements.each do |element| %>
     <%= render partial: "/pulitzer/#{element.class.table_name.gsub(/pulitzer_/,'')}/show_wrapper", locals: { element: element } %>
   <% end %>
   <div class="pulitzer-row">
     <%= render partial: 'pulitzer/versions/form', locals: { version: version } %>
   </div>
   <div class="preview-link">
-    <% if @preview_path %>
+    <% if @preview_path && !version.empty_required_content_elements? %>
```

Comment:
@gomayonqui I think you need `version.required_partials?` too.
maybe you could create a new method `Version#publishable?` and have it check both things, then it would be a cleaner conditional here.